src/App.tsx

```diff
@@ -1,222 +1,273 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+
+import classNames from 'classnames';
 import { AuthContext } from './components/Auth/AuthContext';
+import {
+  getTodos, sendTodos, deleteTodos, patchTodos,
+} from './api/todos';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Todo } from './types/Todo';
+import { SendedTodo } from './types/SendedTodo';
+
+const positionValues = ['All', 'Active', 'Completed'];
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
 
+  const [newTodo, setNewTodo] = useState<SendedTodo | null>(null);
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [position, setPosition] = useState('all');
+  const [currTodoId, setCurrTodoId] = useState(0);
+  const [hidden, setHidden] = useState(true);
```

Comment:
```suggestion
  const [isHidden, setIsHidden] = useState(true);
```
if it's boolean it should be like that
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig